Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -893,12 +904,67 @@ struct AttributeData {
                         break;
                     case VertexAttribute.BlendIndices:
                         attributes.JOINTS_0 = accessorId;
+                        accessor.componentType = GLTFComponentType.UnsignedShort;
                         break;
                     default:
                         throw new ArgumentOutOfRangeException();
                 }
             }
 
+            // Add skin
+            if (uMesh.bindposes != null && uMesh.bindposes.Length > 0)
+            {
+                Skin skin = m_SkinMap[uMesh];
+                Transform[] uBones = m_uBones[m_Skins.IndexOf(skin)];
+            
+                var accessor = new Accessor {
+                    byteOffset = 0,
+                    componentType = GLTFComponentType.Float,
+                    count = uBones.Length,
+                    typeEnum = GLTFAccessorAttributeType.MAT4
+                };
+                
+                var accessorId = AddAccessor(accessor);
+                skin.inverseBindMatrices = accessorId;
+                
+                List<uint> joints = new List<uint>();
+                for (int i = 0; i < uBones.Length; i++)
+                {
+                    uint boneIdx = (uint) m_Nodes.FindIndex(node => node.name == uBones[i].name);
+                    joints.Add(boneIdx);
+                }
+
+                skin.joints = joints.ToArray();
+                
+                Profiler.BeginSample("BindPoseMatricesJobSchedule");
+                var bindPoseCount = uMesh.bindposes.Length;
+                var matrices = new NativeArray<float4x4>(bindPoseCount, Allocator.TempJob);
+                var resultMatrices = new NativeArray<float4x4>(bindPoseCount, Allocator.TempJob);
+
+                for (int i = 0; i < bindPoseCount; i++)
+                {
+                    matrices[i] = uMesh.bindposes[i];
+                }
+                
+                var job = new ExportJobs.ConvertMatrixJob {
+                    input = matrices,
+                    output = resultMatrices
+                }.Schedule(bindPoseCount, k_DefaultInnerLoopBatchCount);
+                Profiler.EndSample();
+                while (!job.IsCompleted) {
+                    await Task.Yield();
+                }
+                Profiler.BeginSample("BindPoseMatricesJobSchedule");
+                job.Complete(); // TODO: Wait until thread is finished

```

Comment:
TODO comment is not necessary, since the job is awaited for in the while loop above